air/src/execution.rs

```diff
@@ -0,0 +1,23 @@
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub struct ExecutionOptions {
+    max_cycles: Option<u32>,
+    expected_cycles: u32,
+}
+
+impl Default for ExecutionOptions {
+    fn default() -> Self {
+        ExecutionOptions {
+            max_cycles: None,
+            expected_cycles: 64,
+        }
+    }
+}
+
+impl ExecutionOptions {
+    pub fn new(max_cycles: Option<u32>, expected_cycles: u32) -> Self {
+        ExecutionOptions {
+            max_cycles,
+            expected_cycles,
+        }
+    }
```

Comment:
Should we do some validation here? For example, if `max_cycles` is provided, it probably shouldn't be smaller than `expected_cycles`.